CMakeLists.txt

```diff
@@ -24,6 +24,7 @@ option(MANIFOLD_USE_CUDA OFF)
 option(MANIFOLD_PYBIND ON)
 option(MANIFOLD_CBIND OFF)
 set(MANIFOLD_PAR "NONE" CACHE STRING "Parallel backend, either \"TBB\" or \"OpenMP\" or \"NONE\"")
+set(MANIFOLD_FLAGS -O3)
```

Comment:
I didn't think so, but I noticed my mac was really slow to run the tests (10-50x) and I realized this wasn't being added to the compile flags. Adding this fixed it, and I figured it can't hurt. Not sure why Clang wasn't adding it automatically for a Release build though.

Reply:
Weird, cmake should be adding this to the `CXX_FLAGS`. Anyway this should be fine, I don't think there are many people building this in debug mode :)